Sources/Rendering/Core/InteractorObserver/index.d.ts

```diff
@@ -0,0 +1,38 @@
+import vtkRenderWindowInteractor from '../../../Rendering/Core/RenderWindowInteractor';
```

Comment:
here is my WIP of InteractorObserver d.ts https://ctxt.io/2/AADgyD48Fg

Reply:
That link is IP restricted for me

Reply:
yeah it should be `import vtkRenderWindowInteractor from "../RenderWindowInteractor";`